metaflow/plugins/aws/batch/batch_decorator.py

```diff
@@ -323,7 +323,7 @@ def _wait_for_mapper_tasks(self, flow, step_name):
         to terminate immediately, and possibly prematurely.
         """
         from metaflow import Step  # avoid circular dependency
-
+        TIMEOUT = 600
         last_completion_timeout = time.time() + 600  # wait for 10 minutes
```

Comment:
I would use TIMEOUT here (instead of 600).

Reply:
ah sorry, fixed now.